### ISSUE TITLE
Issue downloading package

### ISSUE THREAD

**Issue by elevendoky** [kevinblighe/cytofNet#17]
Hello I am trying to download your package to analyze flow data. When I try to download the Bioconductor way I get an error saying `Bioconductor version 3.11 (BiocManager 1.30.10), R 4.0.2 (2020-06-22)
Installing package(s) 'scDataviz'
Warning message:
package ‘scDataviz’ is not available (for R version 4.0.2) `
However when I try to instal the development version for github I get this error
```
`✓  checking for file ‘/private/var/folders/qs/wqy6rgt12z1dzdt0042hsp3r0000gs/T/RtmpvDOkp7/remotes119f3195e80e6/kevinblighe-scDataviz-c18feca/DESCRIPTION’ ...
─  preparing ‘scDataviz’:
✓  checking DESCRIPTION meta-information ...
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘scDataviz_1.1.0.tar.gz’
   
* installing *source* package ‘scDataviz’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: (converted from warning) package ‘S4Vectors’ was built under R version 4.0.3
Execution halted
ERROR: lazy loading failed for package ‘scDataviz’
* removing ‘/Library/Frameworks/R.framework/Versions/4.0/Resources/library/scDataviz’
Error: Failed to install 'scDataviz' from GitHub:
  (converted from warning) installation of package ‘/var/folders/qs/wqy6rgt12z1dzdt0042hsp3r0000gs/T//RtmpvDOkp7/file119f3b60890d/scDataviz_1.1.0.tar.gz’ had non-zero exit status`
```
Any idea how to fix this? Thanks!

**Comment by kevinblighe** [comment]
Hey elevendoky, it seems that you have Bioconductor v3.11 software installed; however, scDataviz was only officially released with Bioconductor 3.12. There are 2 possible solutions, I think:

- update your pre-existing core packages with `BiocManager::install(version = '3.12')` or `BiocManager::install()`


- downlaod the tar.gz file from https://bioconductor.org/packages/release/bioc/html/scDataviz.html and try to install manually via `install.packages()`

**Comment by kevinblighe** [comment]
If this does not resolve the issue, please re-open this question.

Kind regards,
Kevin
